cli/tests/functional/pom.xml

```diff
@@ -97,6 +97,7 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
                 <configuration>
+                    <skipTests>true</skipTests>
```

Comment:
I suppose you'll fix this in a different PR right ?

Reply:
Yes, it is there https://github.com/oracle/helidon-build-tools/pull/753